tests/sentry/integrations/vsts/test_integration.py

```diff
@@ -176,13 +176,47 @@ def test_build_integration(self):
             VSTSIdentityProvider.oauth_scopes
         )
 
-    def test_build_integration__create_subscription_error(self):
+    def test_build_integration__create_subscription_forbidden(self):
```

Comment:
Does it make sense to split these out into a separate class rather than use the `__` syntax?

Reply:
I can split them up in a follow up pull, the test case is getting a bit long.